src/containers/Column/Column.js

```diff
@@ -1,17 +1,26 @@
-import React, { Component } from 'react'
+import React, { Component, PropTypes } from 'react'
 import { connect } from 'react-redux'
 import { Container, Grid } from 'semantic-ui-react'
 
+import { removeColumn } from '../../actions/columns'
 import { showDetails } from '../../actions/details'
 import Company from '../../components/Company'
 import Filter from '../../components/Filter'
 import Sort from '../../components/Sort'
+import Actions from './Actions'
 import './Column.css'
 
 class Column extends Component {
   render () {
+    const actions = [
+      {
+        onClick: this.props.closeThisColumn,
+        text: 'Close Column'
+      }
+    ]
     return (
       <Container>
+        <Actions actions={actions} />
```

Comment:
Lots of actions in one line... but I can't really think of a better alternative word to use, I know saw Trello uses List actions, and I thought of maybe tasks but doesn't really effect the code.